git-credential-oauth.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: git-credential-oauth
+  version: 0.13.4
+  epoch: 0
+  description: "A Git credential helper that securely authenticates to GitHub, GitLab and BitBucket using OAuth"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - git
+
+environment:
+  environment:
+    CGO_ENABLED: 0
```

Comment:
is the binary linking dynamically if we don't pass this? A little more context around this would be useful? We generally try to avoid passing CGO_ENABLED?

Reply:
I typically set this out of habit. I have looked through many of the other Go packages and noticed that it is also not set. So I have no issue with removing this